docs/send-data/kubernetes/collecting-kubernetes-metrics.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
I wonder if we shouldn't just merge this document with the application metrics one and call it `Collecting Metrics`. This one is pretty short and light on content.